index.html

```diff
@@ -229,9 +229,11 @@
                     <atomic-field-condition must-match-sourcetype="Salesforce">
                       <atomic-result-badge label="Salesforce" class="salesforce-badge"></atomic-result-badge>
                     </atomic-field-condition>
-                    <atomic-field-condition class="field" if-defined="language">
```

Comment:
I think it would be better to keep the field condition on `language`, since it makes not sense to display the badge if there's no language 🙂

Reply:
Thanks! I'll tend to that.